lib/bot.js

```diff
@@ -40,7 +41,10 @@ function Bot(options) {
 Bot.prototype.attachListeners = function() {
   this.client.addListener('registered', function(message) {
     logger.debug('Registered event: ', message);
-  });
+    this.autoSendCommands.forEach(function(element) {
```

Comment:
This'll crash the bot if you don't provide an `autoSendCommands`-array, can fix it by changing line 28 to
```js
this.autoSendCommands = options.autoSendCommands || [];
```

so it defaults to an empty array if it doesn't exist.